book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(date, person)
+    Rental.new(date, person, self)
+  end
```

Comment:
- [ ] The arragnement of arguments here is wrong, `self` should come before `person`. Kindly resolve this.

  ```
  def add_rental(date, person)
    Rental.new(date, self, person)
  end
  ```

Reply:
The method has been updated.